### PR TITLE
Android build docs should include 39.0 command

### PR DESCRIPTION
**Pull request by chadlinden**
The current public docs do not include build instructions for React Native Android version 39.

### CODE REVIEW

**Comment by grabbou** [comment]
Documentation is versioned on the website and release specific, having said that - I am not sure if mentioning React version is required.

**Comment by chadlinden** [comment]
The docs for 39.x http://facebook.github.io/react-native/releases/0.39/docs/signed-apk-android.html states: `$ react-native run-android --variant=release `

File: docs/SignedAPKAndroid.md [hunk 1]
```diff
@@ -90,7 +90,13 @@ Before uploading the release build to the Play Store, make sure you test it thor
 $ react-native run-android --variant=release
 ```
 
-Note that `--variant=release` is only available if you've set up signing as described above.
+Note that `--variant=release` is only available if you've set up signing as described above, and are using React Native 40.0
```
Comment (mkonicek) [inline docs/SignedAPKAndroid.md]: nit: period

File: docs/SignedAPKAndroid.md [hunk 2]
```diff
@@ -90,7 +90,13 @@ Before uploading the release build to the Play Store, make sure you test it thor
...
+
+To test the app build using React Native versions 39 and below use the command:
```
Comment (mkonicek) [inline docs/SignedAPKAndroid.md]: nit: built

**Comment by mkonicek** [comment]
Agree with @grabbou - no need to discuss different versions - the docs themselves are versioned.

> The docs for 39.x http://facebook.github.io/react-native/releases/0.39/docs/signed-apk-android.html states: $ react-native run-android --variant=release

This should be fixed in the 0.39 branch.

As an aside, `--configuration=release` and ` --variant=release` look pretty much the same. Not sure why we (I?) accepted that change :)

**Comment by mkonicek** [comment]
nit: Your PR summary contains a lot of irrelevant stuff (the template). This summary is going to become a commit message when PRs get merged.

**Comment by hramos** [comment]
@chadlinden have you signed the CLA yet?

**Comment by hramos** [comment]
Thanks for signing the CLA, without that I was unable to look at the PR. So I find myself in agreement with others here - the docs are versioned and it seems unnecessary to mention versions here.

**Comment by chadlinden** [comment]
Makes sense not to include a 0.39 on 0.40, but the docs for 0.39 are inaccurate in instructing the use of `react-native run-android --variant=release` which should be `react-native run-android --configuration=release`   

So, perhaps the PR should be tagged to 0.39 ?

**Comment by hramos** [comment]
If the new copy should not be displayed on 0.40 and up, then yes, the PR should be opened against the `0.39-stable` branch, as well as a new PR for every other version where this needs to be noted. We generally advise against opening PRs against anything but master but this is one situation where it is needed in order to make sure the change is only reflected in older docs.

**Comment by mkonicek** [comment]
Closing, see the discussion above.

**Pull request closed without merging**